webgui/scripts/codecompass/view/component/Text.js

```diff
@@ -427,7 +418,7 @@ function (declare, domClass, dom, style, query, topic, ContentPane, Dialog,
       //--- Right click ---//
 
       if (event.button === 2)
-        buildContextMenu(pos, this._fileInfo, this._contextMenu);
+        buildContextMenu(pos, urlHandler.getFileInfo(), this._contextMenu);
```

Comment:
If changes of `loadFile()` function are reverted then this modification is not necessary.